### DISCUSSION TITLE
Your account has been flagged #2081439

### DISCUSSION THREAD

**Discussion by summerflowers** [community/community#52383] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hi Dear,

Hope this message find you are doing well and having happy days with your family :)

2 weeks ago, I reporetd the problem of "Your account has been flagged" via github support, but I did not get any answer yet, so I think may be I can ask a question in here? :)

This is my original report:

_Today I found "Your account has been flagged". Sir, I am a coder more than 20 years, I did not publish any project on github, I just forked projects that I loved, I am not a spam, I never say anything on github, I just use my github account to fork greate projects for myself, I cannot understand why "Your account has been flagged", would you please tell me more details so I can correct the problem? 
Sir, If you tell me what's the problem in my account, I'd like correct it, if you think I should not fork projects on github, that's okay, I will delete all forked project on github,  :)_

Sorry for create this ticket in here , I just want to know how to avoid this kind of problem, once I know the reason I will correct the problem on my side :)

Thanks, have a happy day with your family :)

**Comment by cehfisher** [comment]
Hi @summerflowers - It looks like your issue was mistakenly added to the accessibility discussion board. @queenofcorgis could you please move this to the correct category? Thanks!

**Comment by queenofcorgis** [comment, marked as answer]
Hi @summerflowers, 

Thank you for your patience. The community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. We appreciate your understanding! 

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!
